### PR TITLE
Jacob branch - PR for TODO # 2

### PR DESCRIPTION
**Pull request by davasorus**
PR for  #31

### CODE REVIEW

**Comment by davasorus** [comment]
@DeWolfe6 I'll take a look and see what she do

File: Controllers/softwareController.cs [hunk 1]
```diff
@@ -54,9 +84,65 @@ public async Task<IHttpActionResult> GetString()
...
+                mspClient, cadClient32, cadClient64, cadIncObs64
+            };
+
+            List<string> custFilesAddon = new List<string>()
+            {
+            };
+
+            foreach (var file in filePreReq)
+            {
+                if (filePreReq.Equals(custFilesPre))
```
Comment (davasorus) [inline Controllers/softwareController.cs:112]: you may want to change the filePreReq to file.

File: Controllers/softwareController.cs [hunk 2]
```diff
@@ -54,9 +84,65 @@ public async Task<IHttpActionResult> GetString()
...
+                else
+                {
+                    tupleList.Add(new tupleData()
+                    { responseCode = "400 Bad Request", message = $"{custFilesPre} not found on machine" });
+                }
+            }
+
+            foreach (var file in fileAddOn)
+            {
+                if (fileAddOn.Equals(custFilesAddon))
```
Comment (davasorus) [inline Controllers/softwareController.cs:125]: "" as above

File: Controllers/softwareController.cs [hunk 3]
```diff
@@ -54,9 +84,65 @@ public async Task<IHttpActionResult> GetString()
...
+                {
+                    tupleList.Add(new tupleData() { responseCode = "200 OK", message = $"{custFilesAddon} found on machine" });
+                }
+                else
+                {
+                    tupleList.Add(new tupleData() { responseCode = "400 Bad Request", message = $"{custFilesAddon} not found on machine" });
+                }
+            }
+
+            foreach (var file in fileClient)
```
Comment (davasorus) [inline Controllers/softwareController.cs:135]: "" as the first comment

Comment (davasorus) [reply Controllers/softwareController.cs:135]: `if (fileClient.Equals(custFilesClient)) ` -> `if (file.Equals(custFilesClient))`

File: Controllers/softwareController.cs (hunk 2 above)
Comment (davasorus) [reply Controllers/softwareController.cs:125]: `if (fileAddOn.Equals(custFilesAddon))` -> `if (file.Equals(custFilesAddon))`

File: Controllers/softwareController.cs (hunk 1 above)
Comment (davasorus) [reply Controllers/softwareController.cs:112]: `if (filePreReq.Equals(custFilesPre))` -> `if (file.Equals(custFilesPre))`

File: Controllers/softwareController.cs [hunk 4]
```diff
@@ -65,13 +151,11 @@ public async Task<IHttpActionResult> GetStringbyID()
...
         public async Task<IHttpActionResult> GetInstalledSoftware()
         {
-            tupleData tupleOldUpdaterSuccess = new tupleData { responseCode = "200 OK", message = "New World Automatic Updater found on machine" };
-
             List<tupleData> tupleList = new List<tupleData>();
 
             List<string> knownSoftwareList = new List<string>()
         {
-            sqlCE3532Name,sqlCE3564Name, sqlCE4064Name, nwpsGis32Name, nwpsGis64Name, sql2008Clr32Name, sql2008Clr64Name,
+            sqlCE3532Name,sqlCE3564Name, sqlCE4064Name, nwpsGis32Name, nwpsGis64Name,nwpsUpdateName, sql2008Clr32Name, sql2008Clr64Name,
```
Comment (davasorus) [inline Controllers/softwareController.cs:158]: if you could remove `nwpsUpdateName,` I would appreciate it

File: Controllers/softwareController.cs [hunk 5]
```diff
@@ -86,7 +170,7 @@ public async Task<IHttpActionResult> GetInstalledSoftware()
                 }
                 else
                 {
-                    loggingClass.logEntryWriter($"{s} not found on machine", "error");
+                    loggingClass.logEntryWriter($"{s} not found on machine", "Error");
```
Comment (davasorus) [inline Controllers/softwareController.cs:173]: could you change `"Error"` to `"error"`

**Review by DeWolfe6 — COMMENTED** [review]
jeanathon

**Pull request merged**